open-authoring/PRPlugin.tsx

```diff
@@ -4,15 +4,19 @@ import PrIconSvg from '../public/svg/pr-icon.svg'
 import { PRModal } from './PRModal'
 import { DesktopLabel } from '../components/ui/inline/DesktopLabel'
 import { ToolbarButton } from '../components/ui/inline/ToolbarButton'
+import { isGithubTokenValid } from './github/api'
+import OpenAuthoringError from './OpenAuthoringError'
 
 interface PullRequestButtonOptions {
   baseRepoFullName: string
   forkRepoFullName: string
+  sendErrorToModal?
 }
 
 export const PRPlugin = (
   baseRepoFullName: string,
-  forkRepoFullName: string
+  forkRepoFullName: string,
+  sendErrorToModal?: (error: any) => void
```

Comment:
I'd change this to something like `onError`. It shouldn't need to know the specifics on how it will be used/consumed